ossdbtoolsservice/language/completion_refresher.py

```diff
@@ -34,7 +34,10 @@ class CompletionRefresher:
     completion suggestions in a background thread.
     """
 
-    refreshers = OrderedDict()
+    refreshers = {
+        PG_PROVIDER_NAME: OrderedDict(),
+        MYSQL_PROVIDER_NAME: OrderedDict()
```

Comment:
Can we use the same refresher for Mariadb and Mysql?
Suggestions?

Reply:
Will have to investigate further for Mariadb. Making a note for it.